net/respondd/src/respondd.c

```diff
@@ -361,13 +417,132 @@ static void serve(int sock) {
 	}
 
 	if (output) {
-		if (sendto(sock, output, output_bytes, 0, (struct sockaddr *)&addr, addrlen) < 0)
+		if (sendto(sock, output, output_bytes, 0, addr, sizeof(struct sockaddr_in6)) < 0)
```

Comment:
glibc uses the `__attribute__((__transparent_union__))` extension to make socket functions work with any sockaddr type, so no warning is shown. This is neither standard-compliant nor do other libcs like musl implement it.